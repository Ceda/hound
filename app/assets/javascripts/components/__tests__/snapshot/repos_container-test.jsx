import React from 'react';
import renderer from 'react-test-renderer';

import ReposContainer from '../../repos_container.js';

const Hound = window.Hound = global.Hound = {
  settings: {
    placeholder: "meh"
  }
};

it('renders appropriately', () => {
  const onRefreshClicked = jest.genMockFunction();

  const component = renderer.create(
    <ReposContainer
      authenticity_token={"csrf_token"}
      has_private_access={false}
      userHasCard={false}
    />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
