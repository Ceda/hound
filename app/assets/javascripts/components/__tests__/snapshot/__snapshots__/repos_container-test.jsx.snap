exports[`test renders appropriately 1`] = `
<div>
  <div
    className="repo-tools">
    <div
      className="repo-tools-search">
      <input
        className="repo-search-tools-input"
        onChange={[Function]}
        placeholder={undefined}
        type="text" />
    </div>
    <div
      className="repo-tools-private">
      <form
        action="/auth/github?access=full"
        method="post">
        <button
          className="repo-tools-private-button"
          onClick={undefined}
          type="submit">
          <span>
            Include private repos
          </span>
        </button>
      </form>
    </div>
    <div
      className="repo-tools-refresh">
      <button
        className="repo-tools-refresh-button"
        disabled="disabled"
        onClick={[Function]}>
        <span />
      </button>
    </div>
  </div>
  <div
    className="repos-syncing">
    <div
      className="dot" />
    <div
      className="dot" />
    <div
      className="dot" />
  </div>
</div>
`;
